shortcuts.js

```diff
@@ -0,0 +1,17 @@
+import { button } from './index.js';
+
+export const buttons = {
+    'button': button.buttonSecondary,
```

Comment:
Secodary? 🤔

Reply:
It is apparently a default button. https://github.com/fabric-ds/css/blob/next/src/components/button.css#L3